packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -106,16 +106,25 @@ class CupertinoDialog extends StatelessWidget {
 /// An iOS-style alert dialog.
 ///
 /// An alert dialog informs the user about situations that require
-/// acknowledgement. An alert dialog has an optional title and an optional list
-/// of actions. The title is displayed above the content and the actions are
-/// displayed below the content.
+/// acknowledgement. An alert dialog has an optional title, optional content,
+/// and an optional list of actions. The title is displayed above the content
+/// and the actions are displayed below the content.
+///
+/// This dialog styles its title and content to match the standard iOS title
+/// and content dialog text style. These default styles can be overridden by
+/// explicitly defining [TextStyle]s for [Text] widgets that are part of the
+/// title or content.
+///
+/// To display action buttons that look like standard iOS dialog buttons,
+/// provide [CupertinoDialogAction]s for the 'actions' given to this dialog.
```

Comment:
nit: backticks for parameters that aren't also fields. Square brackets for parameters that are also fields. Normal quotes for things that aren't code.